GameData/KerbalismConfig/System/Habitat.cfg

```diff
@@ -213,7 +213,7 @@
   RESOURCE
   {
 	name = Atmosphere
-	amount = 330		 // Kerbals suit up in a pressurized environment so they start pressurized.
+	amount = 0		 // Kerbals suit up in a pressurized environment so they start pressurized.
```

Comment:
The comment is now wrong, you should change it to something like
// Atmosphere will be transferred upon EVAing

Also, this change will be an issue if an EVA Kerbal is spawned without EVAing, this should be verified. I can think of at least one potential case : launching in an external command seat. Theoretically, a mod could also be doing this, but right now I can't think of any that does.